composer.json

```
@@ -23,7 +23,7 @@
     "require": {
         "php": "^8.0",
         "cycle/database": "^2.1",
-        "yiisoft/rbac": "dev-master"
+        "yiisoft/rbac": "dev-165-improve-perfomance"
```

```suggestion
        "yiisoft/rbac": "dev-master"
```

Create issue about changing version rbac.

---

It's already created - #64.